lib/jsdom/living/nodes/Element-impl.js

```diff
@@ -115,8 +114,8 @@ class ElementImpl extends NodeImpl {
     }
 
     // update classList
-    if (name === "class") {
-      resetDOMTokenList(this.classList, value);
+    if (name === "class" && this._classList !== undefined) {
+      this._classList._attrModified();
```

Comment:
It looks like _attrModified shouldn't be "private". Picture a world where we move all our underscore-prefixed stuff to actual private fields/methods, and this doesn't work.

Reply:
Changed.